data/wbm/compile_wbm_test_set.py

```diff
@@ -407,8 +407,7 @@ def fix_bad_struct_index_mismatch(material_id: str) -> str:
     df_wbm[cols].reset_index().to_json(f"{WBM_DIR}/{today}-wbm-{fname}.json.bz2")
 
 
-# %%
-# df_summary and df_wbm formulas differ because summary formulas are reduced while
+# %% df_summary and df_wbm formulas differ because summary formulas are reduced while
```

Comment:
<img width="1046" alt="image" src="https://github.com/user-attachments/assets/5d88f2c7-4488-45c9-9842-07c56dc39e92">
splitting comments like this doesn't strike me as best practice as it hides the starts of the text from the cell block in jupyter

Reply:
i don't see that as a negative. i think i prefer to look at code over comments anyway to recognize cell<->source association